### ISSUE TITLE
Waiting for dependencies until eternity

### ISSUE THREAD

**Issue by orhun** [getanteon/anteon-helm-charts#11]
Hey, I followed the instructions in this repo to deploy Anteon as a helm chart. However, as you can see from the screenshot below, most of the pods are stuck in `Init:0/1` status and the logs just say `[check-dependencies]  waiting for dependencies`.

![image](https://github.com/user-attachments/assets/6104a9c2-09d3-49c4-8ff2-ccc5f8c1112d)

I waited for some time to see if they are going to run but no luck.

I can access the frontend but it is stuck on this page:

![image](https://github.com/user-attachments/assets/2bb42dbc-b18e-4a9c-95d7-0c8548958c2b)

Is there a way of figuring out which pod is it waiting? How can I debug this?

**Comment by orhun** [comment]
Just realized Nginx deployment has the following logs:

![image](https://github.com/user-attachments/assets/18d6d2ae-4526-49de-b79e-5f4661d04e7a)

**Comment by fatihbaltaci** [comment]
Hi @orhun, which StorageClass do you use in your Kubernetes cluster? It looks like PersistentVolumes are stuck in a Pending state. 

```bash
kubectl get storageclass
```

**Comment by orhun** [comment]
Good catch! I didn't realize I had no storages set up for microk8s so I followed [this guide](https://microk8s.io/docs/addon-hostpath-storage) to enable it. Now everything works fine:

![image](https://github.com/user-attachments/assets/6669802a-b617-4109-81f9-b5bc6dd884e1)

We should probably mention that a `storageclass` is required in order for Anteon to run.
